.github/workflows/tests.yaml

```diff
@@ -16,7 +16,7 @@ on:
       git-ref:
         description: 'Which git ref to checkout'
         type: string
-        default: 'main'
+        default: ${{ github.ref }}
```

Comment:
This is `github.ref_name`

Reply:
You can browse the `github` context in this workflow run output https://github.com/szepeviktor/byte-level-care/actions/runs/7585113229/job/20660246214

Reply:
from https://docs.github.com/en/actions/learn-github-actions/contexts i think `github.ref` points to the same and is more descriptive?  Any reason to prefer `ref_name`?

This is moved to https://github.com/n0-computer/iroh/pull/1971 btw.  Let's merge main into this branch here once that's merged.